utils/s2n_asn1_time.c

```diff
@@ -68,166 +69,151 @@ static inline void get_current_timesettings(long *gmt_offset, int *is_dst) {
  * just do a character at a time and change the state per character encountered.
  * when finished the above time structure should be filled in along with some
  * crazy timezone info we'll need shortly afterwards.*/
-static S2N_RESULT process_state(parser_state *state, char current_char, struct parser_args *args) {
-    switch (*state) {
-        case ON_YEAR_DIGIT_1:
```

Comment:
Did you make any changes here other than fixing the indentation? The diff is pretty difficult to read.

Reply:
Oh, crap, clang-format got ahold of it. Let me fix that.

Reply:
And no, it's a one-liner otherwise

Reply:
Fixed.